internal/security/proxy/certs.go

```diff
@@ -22,21 +22,19 @@ import (
 	"io/ioutil"
 	"net/http"
 	"strings"
-
-	"github.com/edgexfoundry/edgex-go/internal"
 )
 
 type CertificateLoader interface {
 	Load() (*CertPair, error)
 }
 
 type certificate struct {
-	client    internal.HttpCaller
+	client    Requestor
 	certPath  string
 	tokenPath string
 }
 
-func NewCertificateLoader(r internal.HttpCaller, certPath string, tokenPath string) CertificateLoader {
```

Comment:
@tingyuz it looks like this is replacing usage of `internal.HttpCaller` with `Requestor`, if that's the case we should do that everywhere, as it is we still have duplication between the HttpCaller and Requestor interfaces - we should only have one of those in the codebase

Reply:
I just did a review and found that other components such as internal\core\command is still referring it so it may not be a good idea to delete this file. I will prefer to leave this decision to the author of last component that refers to it.

Reply:
I would argue that the burden for consolidating the usages falls on you and/or @tsconn23 because y'all's PR is the one that introduced the duplicated code and I specifically asked that it be resolved in the PR but had to agree for the issue to be solved later so here we are

Reply:
@anonymouse64 @tingyuz This WAS removed. Why it's back, I have no idea.
https://github.com/edgexfoundry/edgex-go/pull/1660

Reply:
@tsconn23 my concern was resolved - see later comments on the PR, I was objecting to the duplicated Requestor and HttpCaller